dockerfiles/hpcc/templates/_helpers.tpl

```diff
@@ -30,7 +30,9 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{- /* Translate a port list to a comma-separated list */ -}}
+{{/*
+Translate a port list to a comma-separated list
+*/}}
```

Comment:
trivial: should other template functions below be changed to same comment format ?

Reply:
Yes, eventually